jme3-core/src/main/resources/Common/MatDefs/Light/PBRLighting.frag

```diff
@@ -134,10 +134,12 @@ void main(){
         vec4 albedo = Color;
     #endif
 
+    //ao in r channel, roughness in green channel, metallic in blue channel!
+    vec3 metallicRoughnessAoValue = vec3(1.0, 1.0, 0.0);
```

Comment:
could you call it something like aoRoughnessMetallicValue to respect the order of the channels?